composer.json

```
@@ -137,6 +137,7 @@
                 "Config_sync_test core constraint": "https://gist.githubusercontent.com/tadean/602c6412fbbdcc16a2d8a0fb58642f9e/raw/2ef503a4c0ae7d459f7c789d6ad3720d92945880/config_sync_test_core_requirement_issue.patch"
             },
             "drupal/core": {
+                "Views Date Filter Datetime Granularity Option (2868014)" : "https://www.drupal.org/files/issues/2022-10-10/2868014-124.patch",
```

I think we should try the latest version of the patch.  There are several related core issues that we could try to include patches for as well but I think if the latest version of the patch for this core issue works, that would be OK for now.
```suggestion
                "Views Date Filter Datetime Granularity Option (2868014)" : "https://www.drupal.org/files/issues/2022-11-16/2868014-127.datetime-granularity.patch",
```